quantum/process_keycode/process_tap_term_keys.c

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 Vladislav Kucheriavykh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include "quantum.h"
+#include "process_tap_term_keys.h"
+
+#ifndef TAP_TERM_INCREMENT
+#    define TAP_TERM_INCREMENT 5
+#endif
+
+static void tapping_term_report(void) {
+    char display[8];
+
+    snprintf(display, 8, "%d", g_tapping_term);
```

Comment:
Just for consistency.
```suggestion
    snprintf(display, sizeof(display), "%d", g_tapping_term);
```